tree/dataframe/test/datasource_ntuple.cxx

```diff
@@ -267,6 +267,67 @@ TEST_F(RNTupleDSTest, ChainMT)
    ChainTest(fNtplName, fFileName);
 }
 
+TEST_F(RNTupleDSTest, AlternativeColumnTypes)
+{
+   auto df = ROOT::RDF::Experimental::FromRNTuple(fNtplName, fFileName);
+
+   auto usingDouble = df.Define("nJets", [](ROOT::RVec<double> jets) { return jets.size(); }, {"jets"})
```

Comment:
```suggestion
   auto usingDouble = df.Define("nJets", [](const ROOT::RVec<double> &jets) { return jets.size(); }, {"jets"})
```

I would try to use `const &` everywhere in function arguments passed to RDF API. That being said, we may want to add an extra ad hoc test case for when the user wants to modify the values of an input column. i.e. with a first `Define` call that takes the input value as `ROOT::RVec<double> &`, modfies it, then in a second `Define` or `Filter` perform some computation or selection based on the modified values. I'll let you decide if you want to add it for this PR, but it can also be added separately

Reply:
Noted! I will address this in a separate PR.